### ISSUE TITLE
[HOME] "It's not about the numbers"

### ISSUE THREAD

**Issue by GeoWoodstockXXI** [kkudlak45/gwxxi#80]
<img width="508" alt="PNG image" src="https://github.com/user-attachments/assets/a73f0127-1b3b-41a0-946d-8a849025a48a">

In addition to these four columns, I want a fifth column on the far left (so the first one) that says ### days until GeoWoodstock XXI with a countdown clock to May 24, 2025.

**Comment by GeoWoodstockXXI** [comment]
We are now up to 19 countries.

**Comment by GeoWoodstockXXI** [comment]
We are now up to 20 countries.

**Comment by GeoWoodstockXXI** [comment]
We are now up to 21 countries. This update needs to go live on the website soon.

**Comment by kkudlak45** [comment]
![Image](https://github.com/user-attachments/assets/f938261c-3323-436d-b0d9-36ea88b5e411)

looks pretty good to me what about you?

**Comment by kkudlak45** [comment]
TODO:
- [ ] add countdown clock
- [ ] make styling not look like shit
- [ ] check responsive behavior (how is my mobile view?)

**Comment by GeoWoodstockXXI** [comment]
Any updates on this?

**Comment by kkudlak45** [comment]
idrly know what you mean by countdown clock. "days until" will auto count

<img width="1548" alt="Image" src="https://github.com/user-attachments/assets/05d01f29-f951-46a2-a138-5071b9576a7c" />

**Comment by kkudlak45** [comment]
didn't mean to close

**Comment by kkudlak45** [comment]
going live shortly. also i added an actual map embed

<img width="1680" alt="Image" src="https://github.com/user-attachments/assets/40a1fb05-d75c-48ac-86ff-9557a9bd6354" />

**Comment by kkudlak45** [comment]
DAMMIT

**Comment by GeoWoodstockXXI** [comment]
Yep. this looks good. just remember to update the will attends during each update.
